### DISCUSSION TITLE
[Enhancement] Heart rate monitor: possibility to set measurement interval without interpolation

### DISCUSSION THREAD

**Discussion by petermolnar** [InfiniTimeOrg/InfiniTime#1740] [Ideas]
I've recently been diagnosed with an irregular heartbeat - something that messes up even high-end medical instrument interpolation. It would be really nice to have a heart rate monitor that shows the actual beats per measured time, or to set the measurement interval maybe even up to multiple minutes to get a better reading.

**Comment by LinuxinaBit** [comment]
There is a pr that should fix general heart-rate measurement first: #1486 
I'm not even sure there is any interpolation, it's just all around bad as of now.
